pkg/cmd/attestation/test/data/sigstore-js-2.1.0-bundle.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
I defer to @williammartin regarding location of testing resources such as this relative to the relevant tests and code under test. 🤷